chia/full_node/mempool.py

```diff
@@ -403,15 +410,39 @@ def create_bundle_from_mempool_items(
                 continue
             try:
                 cost = uint64(0 if item.npc_result.conds is None else item.npc_result.conds.cost)
-                unique_coin_spends, cost_saving, unique_additions = eligible_coin_spends.get_deduplication_info(
-                    bundle_coin_spends=item.bundle_coin_spends, max_cost=cost
-                )
+                if skipped_items >= ELIGIBILITY_SKIP_THRESHOLD:
```

Comment:
I think it would be better to call this by the actual feature, e.g. `DEDUP` rather than the fact that items are *eligible* for said feature. It will become especially confusing when we add more features to the mempool that items may be eligible for